transcriptic/config.py

```diff
@@ -287,6 +287,14 @@ def datasets(self, project_id=None, run_id=None):
             '404': lambda resp: Exception("[404] No run found for ID " + id)
         })
 
+    def get_zip(self, data_id):
+        """Get zip file with given data_id. Returns Python Zipfile"""
+        import zipfile
+        from io import BytesIO
+        route = self.get_route('get_data_zip', data_id=data_id)
+        req = self.get(route, status_response={'200': lambda resp: resp}, stream=True)
```

Comment:
api.get passes on the stream=true parameter to requests.get, so the req.content will be a stream